tox.ini

```
@@ -1,5 +1,5 @@
 [tox]
-envlist = py27, flake8
+envlist = py38, flake8
```

Mopidy supports Python 3.7 too, and that's all you have on the latest Debian stable and thus the latest Raspberry Pi OS, which is an important user group.

```suggestion
envlist = p37, py38, flake8
```